app/models/broadcast_message.py

```diff
@@ -420,6 +441,19 @@ def clear_areas(self, force_override=False):
 
         self._update(**data)
 
+    def get_versions(self):
+        return broadcast_message_api_client.get_broadcast_message_versions(self.service_id, self.id)
+
+    def get_count_of_versions(self):
+        return len(self.get_versions())
+
+    def get_latest_version(self):
+        return (
+            broadcast_message_api_client.get_broadcast_message_versions(self.service_id, self.id)[0]
+            if len(broadcast_message_api_client.get_broadcast_message_versions(self.service_id, self.id)) > 0
```

Comment:
Would this not make identical api calls back to back?  Would it not be better to fetch the result from the API and then make decisions on that result rather than calling get_broadcast_message_versions twice?